### PR TITLE
update import dataset as single page

### PR DESCRIPTION
**Pull request by elliotmessi**

### CODE REVIEW

File: src/locales/modules/dataset.ts [hunk 1]
```diff
@@ -72,6 +72,19 @@ const dataset = {
...
   "dataset.add.form.name.required": { cn: "请输入数据集名称", en: "Dataset Name", },
   "dataset.add.form.type.label": { cn: "导入类型", en: "Type", },
+  "dataset.add.form.label.label": { cn: "导入标注", en: "Type", },
+  "dataset.add.form.newkw.label": { cn: " ", en: " ", },
+  "dataset.add.newkw.asname": { cn: "添加标签", en: "As Keyword", },
+  "dataset.add.newkw.asalias": { cn: "添加为别名", en: "As Alias", },
+  "dataset.add.newkw.ignore": { cn: "忽略此标签", en: "Ignore", },
+  "dataset.add.form.newkws.label": { cn: "添加标签", en: "Add Keywords", },
+  "dataset.add.form.newkw.link": { cn: "去添加>>", en: "Add New Keywords>>", },
+  "dataset.add.form.newkw.tip": { cn: "当导入数据集的标签不属于当前标签时，选择标签合并策略", en: "When ", },
```
Comment (phoenix-xhuang) [inline src/locales/modules/dataset.ts:82]: 这里是未完成？

Comment (elliotmessi) [reply src/locales/modules/dataset.ts:82]: Select keyword merge strategy when keywords of importing dataset not include in your Keyword List

Comment (phoenix-xhuang) [reply src/locales/modules/dataset.ts]: 看着没有加到en代码里面

Comment (elliotmessi) [reply src/locales/modules/dataset.ts]: 加了

File: src/locales/modules/dataset.ts [hunk 2]
```diff
@@ -72,6 +72,20 @@ const dataset = {
...
   "dataset.add.form.type.label": { cn: "导入类型", en: "Type", },
+  "dataset.add.form.label.label": { cn: "导入标注", en: "Type", },
+  "dataset.add.form.newkw.label": { cn: " ", en: " ", },
+  "dataset.add.newkw.asname": { cn: "添加标签", en: "As Keyword", },
+  "dataset.add.newkw.asalias": { cn: "添加为别名", en: "As Alias", },
+  "dataset.add.newkw.ignore": { cn: "忽略此标签", en: "Ignore", },
+  "dataset.add.form.newkws.label": { cn: "添加标签", en: "Add Keywords", },
+  "dataset.add.form.newkw.link": { cn: "去添加>>", en: "Add New Keywords>>", },
+  "dataset.add.form.newkw.tip": { cn: "当导入数据集的标签不属于当前标签时，选择标签合并策略", 
+  en: "Select keyword merge strategy when keywords of importing dataset not include in your Keyword List", },
```
Comment (phoenix-xhuang) [inline src/locales/modules/dataset.ts:83]: Select keyword merging strategy when new keywords are not included in current list

File: src/locales/modules/dataset.ts [hunk 3]
```diff
@@ -72,6 +72,20 @@ const dataset = {
...
+  "dataset.add.label_strategy.include": { cn: "包含标注", en: "Include Labels", },
+  "dataset.add.label_strategy.exclude": { cn: "不包含标注", en: "Exclude Labels", },
+  "dataset.add.label_strategy.ignore": { cn: "忽略新标签和对应标注", en: "Ignore new keywords and relavant labels", },
```
Comment (phoenix-xhuang) [inline src/locales/modules/dataset.ts:86]: Ignore unknown keywords and annotations

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**